server/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const bodyParser= require('body-parser');
+const app = express();
+const port = 1022;
+const Home = require('../database/index');
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost:27017/homes', {useUnifiedTopology: true, useNewUrlParser: true, useCreateIndex: true });
+
+//body parser
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+//static
+app.use(express.static('public'));
+
+//routes
+app.get('/homes', function (req, res) {
+  var homes = Home.find()
+    .then((homes) => {
+      res.send(homes);
+    })
+    .catch((err) => {
+     res.send(err);
```

Comment:
Never send the database error message to the frontend; this is a security concern.

Rather, send a more general "oops, something went wrong!" with a proper error status code.